src/utils/helpers/vaults.ts

```diff
@@ -13,7 +13,7 @@ const getCollateralization = (
   if (tokens.gt(BitcoinAmount.zero()) && btcToCollateralTokenRate.toBig().gt(0)) {
     const tokensAsCollateral = btcToCollateralTokenRate.toCounter(tokens);
     // ray test touch <<
-    return collateral.toBig(0).div(tokensAsCollateral.toBig(0)).mul(100);
+    return collateral.toBig(0).div(tokensAsCollateral.toBig(0));
```

Comment:
![Capture](https://user-images.githubusercontent.com/84005068/186271006-f1b897e3-4119-419e-9f52-f14a1a1c62c1.PNG)
There was a bug that percentages are compared with rates. So I fixed it although it's out of scope of this PR.